Makefile

```diff
@@ -602,6 +602,14 @@ scalaclean:
 	(cd $(ROOTDIR)/scala-package; \
 		mvn clean -P$(SCALA_PKG_PROFILE),$(SCALA_VERSION_PROFILE))
 
+scalatestcompile:
+	(cd $(ROOTDIR)/scala-package; \
```

Comment:
Why the parenthesis?  Also what if cd fails?

Reply:
I mostly left the parenthesis because that is consistent with the rest of the scala make commands and we have been following that since 2452d87538c809509f964d625fbd007debc8224e. They could all stand to be cleaned up, but that should be handled in a separate PR.

When you say cd fails, are you referring to the change directory or continuous deployment? Either way, is there something that should happen when it fails or some greater consequence of it failing? I'm not sure I get the question

Reply:
you might want to use && instead of ; in case the previous command fails.

```
pllarroy@186590d670bd:0:~$ false ; echo keep going
keep going
pllarroy@186590d670bd:0:~$ false && echo keep going
pllarroy@186590d670bd:1:~$
```

Reply:
https://stackoverflow.com/questions/1789594/how-do-i-write-the-cd-command-in-a-makefile

Reply:
Oh. That makes sense. I fixed it